src/Form/NewsletterSubscriptionType.php

```diff
@@ -26,6 +35,25 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('personalDataCollection', AcceptPersonalDataCollectType::class)
         ;
+
+        $builder->addModelTransformer(new CallbackTransformer(
+            function ($data) {
+                return $data;
+            },
+            function ($subscription) {
+                if ($subscription instanceof NewsletterSubscription) {
+                    $existingSubscription = $this->newsletterSubscriptionRepository->findOneNotConfirmedByEmail($subscription->getEmail());
+                    if ($existingSubscription) {
+                        $subscription->setCountry($subscription->getCountry());
+                        $subscription->setPostalCode($subscription->getPostalCode());
```

Comment:
I guess it should be :
```suggestion
                        $subscription->setCountry($existingSubscription->getCountry());
                        $subscription->setPostalCode($existingSubscription->getPostalCode());
```
no ?

Reply:
😱